client/my-sites/patterns/components/section/index.tsx

```diff
@@ -13,13 +16,18 @@ type PatternsSectionProps = {
 
 export const PatternsSection = ( {
 	title,
+	forwardRef,
+	id,
 	description,
 	theme,
 	bodyFullWidth,
 	children,
 }: PatternsSectionProps ) => {
+	const sectionProps = id ? { id } : {};
 	return (
 		<section
+			ref={ forwardRef }
+			{ ...sectionProps }
```

Comment:
Nitpick (non-blocking): Considering the number of allowed props we have right now, I think this is more legible if we define them here explicitely.


```suggestion
			id={ id }
```

Reply:
PatternsSection is used in a few places. It made sense to avoid adding an empty "id" if the prop is not passed.

Reply:
React won't add the prop if it's undefined

Reply:
Oh, TIL!